### ISSUE TITLE
Hello!!

### ISSUE THREAD

**Issue by Yash-Varshney-Creativities** [liament0100/1st-Website#1]
Hi Liament, I'm Yash. I hope you recognized me. You're doing a great job. I saw your 1st, 2nd, 3rd, 4th Site Layout etc. I loved your work. I too am a person who likes web development. I use HTML, CSS, SCSS, Bootstrap, JS, Python etc. I too am like you. A beginner in WebDev. So, I guess we can do some basic stuff at a time? I hope we will learn much more. Can We? I am a small kid only, so, don't think that I'm your client or something...

I hope you would like this. Don't Forget to reply here.

<b><i>Thanks<br><a href="https://yash-varshney.herokuapp.com/">Yash</a></i></b>

**Comment by liament0100** [comment]
Hello yeah how are you, I looked at your work and it’s very professional
and may I say at your age you are destined for a great future 100%. You
will do very well, we can work together on projects. I have an idea in mind
but I need to learn how to code self-learning AI bots I want a chat room
where people can talk to these bots I want them that realistic you can’t
tell the different between human and robot.

**Comment by Yash-Varshney-Creativities** [comment]
Do You Want a Bot like [this?](https://yash-varshney.herokuapp.com/Chat)

**Comment by liament0100** [comment]
Yes exactly one like that but with so many replies to questions humans ask,
I’m thinking building multiple bots with multiple personalities how did you
code that bot ?

**Comment by Yash-Varshney-Creativities** [comment]
I created it with the python backend.

**Comment by Yash-Varshney-Creativities** [comment]
Can we chat on hangouts, skype or telegram, please?
Can we?
I don't think this is the right place. I think this is mostly for program issues and is public.

**Comment by liament0100** [comment]
Nice link me if you can gonna look when get back on computer tomorrow

**Comment by Yash-Varshney-Creativities** [comment]
Are you on hangouts? skype?

I would share a link to join a chat.

**Comment by liament0100** [comment]
Telegram - @mrunknown0100

**Comment by Yash-Varshney-Creativities** [comment]
I sent you a message. 
![image](https://user-images.githubusercontent.com/74306688/111735044-4e515f80-88a1-11eb-94ca-71ce04111316.png)

**Comment by Yash-Varshney-Creativities** [comment]
Hello, Liament
